mappings/net/minecraft/client/gui/hud/DebugHud.mapping

```diff
@@ -0,0 +1,25 @@
+CLASS net/minecraft/class_688 net/minecraft/client/gui/hud/DebugHud
+	FIELD field_2900 client Lnet/minecraft/class_669;
+	FIELD field_2901 renderer Lnet/minecraft/class_679;
+	METHOD <init> (Lnet/minecraft/class_669;)V
+		ARG 1 client
+	METHOD method_2501 renderLeftText ()V
+	METHOD method_2502 interpolateColor (IIF)I
+		ARG 1 color1
+		ARG 2 color2
+		ARG 3 dt
```

Comment:
what is this

Reply:
I have no idea what that means, I just picked it up from [1.16.5](https://github.com/FabricMC/yarn/blob/2eae0214d779c51c4cf065a7f5dce4be266e5f94/mappings/net/minecraft/client/gui/hud/DebugHud.mapping#L31).